pages/pipelines/secrets.md.erb

```diff
@@ -155,3 +155,49 @@ steps:
+
+Next, navigate to the Two-Factor Authentication tab and click it (you may be asked to enter your password in the Confirm Password field).
+<%= image "2fa-3.png", width: 380, height: 238, alt: "Screenshot of Password Confirmation" %>
+<br>
+Enter your Buildkite account password and proceed.
+<%= image "2fa-2.png", width: 850, height: 372, alt: "Screenshot of Accessing Two-Factor Authentication" %>
+
+Click the Setup Two-Factor Authentication button to start securing your Buildkite account. 
+<%= image "2fa-4.png", width: 1172, height: 480, alt: "Screenshot of Setup Button for Two-Factor Authentication" %>
+
+**Step 1: Store recovery codes.**  
+You will need them in case you lose access to your authenticator application to restore access to your account. Use the buttons to either copy the codes to Clipboard or download them as a text file. Keep your recovery codes in a safe digital space or print them out and hide them well. Never share your recovery codes. 
+
+<%= image "2fa-5.png", width: 606, height: 732, alt: "Screenshot of Recovery Codes for Two-Factor Authentication" %>
+
+Once you've saved your recovery codes, click Next.
+
+**Step 2: Configure Authenticator Application**  
+To activate two-factor authentication, scan the barcode that appears in the Buildkite dashboard with the authenticator application of your choice. If you cannot scan the barcode, you can use the secret key below the barcode.
+
+After you've scanned the barcode or activated the authenticator application using the secret key, Buildkite will appear on the list of accounts registered in that application. Your authenticator will provide a new randomly generated six-digit code (your One Time Pass) roughly every 30 seconds. Enter this code into the corresponding field in the Buildkite app and click Activate. 
+
+<%= image "2fa-6.png", width: 604, height: 728, alt: "Screenshot of Barcode and Secret Key for Two-Factor Authenticator" %>
+
+Congratulations! You have now successfully enabled the two-factor authentication for your Buildkite account. This will be confirmed by an 'Enabled' badge next to the Two-Factor Authentication option in your Personal Settings. 
+
+<%= image "2fa-7.png", width: 1172, height: 350, alt: "Screenshot of Two-Factor Authentication Enabled Badge" %>
+
+Next time you try to log into your Buildkite account from a new browser, device, or location, you will be asked to enter the current One Time Password provided by your authentication app.
+
+You can always reconfigure or deactivate the 2FA if you need to. This can be done in the Two-Factor Authentication tab in Personal Settings for your Buildkite account in the dashboard.  
+
+[1Password]: <https://support.1password.com/one-time-passwords/>
+[OTP Auth]: <https://cooperrs.de/otpauth.html>
+[Authy]: <https://authy.com/>
+[Duo Mobile]: <duo.com/product/multi-factor-authentication-mfa/two-factor-authentication-2fa>
```

Comment:
And this needs an `https` and is what is causing the Rspec link error.

(the spell checker is also confused by `Authy`, which needs adding to `vale/vocab.txt`

Reply:
Will deal with duo.com & Authy.